managing_providers/_topics/automation_management_providers.md

```diff
@@ -365,3 +365,174 @@ To use the button to run an Ansible Tower job on a virtual machine:
+
+1. Enable the Embedded Terraform server role.
+2. Add a source control repository that contains your templates.
+3. Add credentials to access the repository.
+4. Create a Service Catalog item with the desired Terraform template.
+
+### Enabling the Embedded Terraform Server Role
+
+In {{ site.data.product.title_short }}, the Embedded Terraform server role is disabled by default. Enable this server role to use Embedded Terraform (OpenTofu).
+
+To enable the Embedded Terraform Server Role, use the following steps:
+
+1. Browse to the settings menu, and click **Configuration** > **Settings**.
+2. Select the desired server under **Zones**.
+3. Set the **Server Role** for **Embedded Terraform** to `On`.
+
+### Verifying the Embedded Terraform worker state
+
+Verify that the Embedded Terraform worker is started to use its features:
+
+1. Browse to the settings menu, then click **Configuration** > **Diagnostics** and click the server that you want to choose.
+2. Click **Roles by Servers** tab.
+
+A table of all workers and their status appears from which you can confirm the state of your Embedded Terraform worker.
+
+### Adding a Template Repository
+
+To enable {{ site.data.product.title_short }} to discover and use your Terraform templates, add a repository to store and manage your templates.
+
+If your repository requires credentials for access, then you need to create SCM credentials. For more information about how to create SCM credentials, see [SCM credentials](../_topics/opentofu_credentials.md#scm).
+
+Use the following steps to add a repository:
+
+1. Browse to the menu and click **Automation > Embedded Terraform > Repositories**.
+
+2. Click **Configuration**, then ![Add New Repository](../images/1862.png) (**Add New Repository**).
+
+3. Provide a Repository Name in the **Name** field.
+
+4. Add a description for the repository in the **Description** field.
+
+5. Add a **URL** and an optional port for the repository.
+
+6. Select the appropriate **SCM Credentials** from the drop-down menu.
+
+7. Provide a branch name in the **SCM Branch** field. This field is optional and default value is set to `master` branch.
+
+8. Click **Save**.
+
+When you save the repository, the Terraform templates are synced, and are available to {{ site.data.product.title_short }}.
+
+### Refreshing Repositories
+
+You can use {{ site.data.product.title_short }} to refresh specific Terraform repositories or all repositories in your inventory to make sure that your templates are up to date.
+
+Use the following steps to refresh a specific repository:
+
+1. Browse to the menu and click **Automation > Embedded Terraform > Repositories**.
+
+2. Click a repository.
+
+3. Click **Configuration**, then ![Refresh this Repository](../images/2003.png) (**Refresh this Repository**).
+
+Alternately, you can refresh some or all of the repositories from the list view:
```

Comment:
Super-minor

```suggestion
Alternatively, you can refresh some or all of the repositories from the list view:
```

Alternately really means like "going back and forth; one after another" vs Alternatively which means "as another option"